apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -386,6 +386,7 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
      */
     useEffect(() => {
         const sortedSchemas = ProfileUtils.flattenSchemas([ ...profileSchemas ])
+            .filter(item => item.name !== "meta.version")
```

Comment:
`meta.version` get this from SCIM_SCHEMA_DICTIONARY

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3122/commits/156699ec6886c2ec6ad68ac4c99f5d2568f88bba